packages/block-library/src/button/edit.js

```diff
@@ -124,19 +165,37 @@ class ButtonEdit extends Component {
 								} }
 							/>
 						</PanelColorSettings>
+						<PanelBody title={ __( 'Link Settings' ) }>
+							<TextControl
+								label={ __( 'Link Rel' ) }
+								value={ rel || '' }
+								onChange={ this.onSetLinkRel }
+							/>
+						</PanelBody>
 					</InspectorControls>
 				</div>
 				{ isSelected && (
-					<form
-						className="block-library-button__inline-link"
-						onSubmit={ ( event ) => event.preventDefault() }>
-						<Dashicon icon="admin-links" />
-						<URLInput
-							value={ url }
-							onChange={ ( value ) => setAttributes( { url: value } ) }
-						/>
-						<IconButton icon="editor-break" label={ __( 'Apply' ) } type="submit" />
-					</form>
+					<URLPopover
+						position={ PopoverPosition }
+						renderSettings={ () => (
+							<ToggleControl
+								label={ __( 'Open in New Tab' ) }
+								onChange={ this.onSetNewTab }
+								checked={ linkTarget === '_blank' }
+							/>
+						) }
+					>
+						<form
+							className="block-library-button__inline-link"
+							onSubmit={ ( event ) => event.preventDefault() }>
+							<Dashicon icon="admin-links" />
```

Comment:
I noticed the other link popovers (in the paragraph and image block don't have this icon). I do like it, but think it's a good idea to make the UI consistent. 

cc: @jasmussen for thoughts on this? (sorry, pinging you a lot today 😄 ).

Reply:
Another great catch, and ping me all you like, it's always a pleasure! 

Yes, this icon is a remnant from the old button UI where it was the only indicator of the field. Let's remove it as part of this unification.

Reply:
UI revamp done in [this commit](https://github.com/WordPress/gutenberg/pull/10128/commits/a519a39c4d6c848f49eb3c441171473225c63ce1), thank you @talldan and @jasmussen ❤️